.github/ISSUE_TEMPLATE/feature_tracking.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Tracking Issue
+about: Collect discussion and track the progress of a shiny new feature
+title: ''
+labels: C-Enhancement, C-Tracking-Issue S-Needs-Triage
+assignees: ''
+---
+
+## What does this new feature do?
+
+**NOTE:** please only create tracking issues for features that have a reasonably clear consensus and implementation plan (although no definite timeline is needed).
```

Comment:
I think this criteria is more suited to "RFC implementation tracking issues". What we discussed previously is more like "generic feature request tracking issues" with rollups of all conversations / designs for a given topic. The former requires consensus (and probably a merged RFC). The latter just requires that one or more people are discussing a feature.

Reply:
Changed to:

> **NOTE:** please only create tracking issues for features if you're interested in tracking that feature's progress until it is released.
This allows users to quickly find and read a single source of truth for the status of a feature they care about.
For feature requests and early design work, please create a [Discussion](https://github.com/bevyengine/bevy/discussions) instead, which can then be linked in a tracking issue.